renovate.json

```diff
@@ -17,13 +17,19 @@
         "ghcr.io/immich-app/immich-machine-learning"
       ],
       "groupName": "immich"
+    },
+    {
+      "matchFileNames": ["charts/*/values.yaml"],
```

Comment:
This doesn't need a filename matcher, any digest update should be a minor version bump.

This package rule is missing the bumpVersions configuration so it actually knows to bump the Chart.yaml version field.